lib/main.js

```diff
@@ -15,6 +16,7 @@ Object.assign(exports, {
 
     // options.loadReason can be install/enable/startup/upgrade/downgrade
     PlacesProvider.links.init();
+    TabTracker.init();
```

Comment:
Can we do that init in ActivityStreams with an optional parameter?
Something like {telemetry: true}, where telemetry is enabled via a pref